action/src/index.ts

```diff
@@ -0,0 +1,63 @@
+import * as core from '@actions/core';
+import * as github from '@actions/github';
+import * as childProcess from 'child_process';
+import * as fs from 'fs';
+
+/**
+ * writeFileToConsole contents of file to console.
+ * @param {string} path - filepath to write to the console
+ */
+function writeFileToConsole(path: string) {
+  try {
+    const data = fs.readFileSync(path, 'utf8');
+    console.log(`${path}: ${data}`);
+  } catch (e) {
+    console.log(`$unable to read {path}, skipping: ${e}`);
+  }
+}
+
+/**
+ * Run a specified command.
+ * @param {string} cmd - command to run
+ */
+function runCmd(cmd: string) {
+  try {
+    childProcess.execSync(cmd);
+  } catch (error) {
+    writeFileToConsole('serverlog_stdout.txt');
+    writeFileToConsole('serverlog_stderr.txt');
+    writeFileToConsole('function_output.json');
+    throw error;
+  }
+}
+
+async function run(): Promise<void> {
+  try {
+    const functionType = core.getInput('functionType');
+    const validateMapping = core.getInput('validateMapping');
+    const source = core.getInput('source');
+    const target = core.getInput('target');
+    const runtime = core.getInput('runtime');
+    const tag = core.getInput('tag');
+
+    // Install conformance client binary.
+    runCmd(
+        'go install github.com/GoogleCloudPlatform/functions-framework-conformance/client');
+
+    // Run the client with the specified parameters.
+    runCmd([
+      `go run github.com/GoogleCloudPlatform/functions-framework-conformance/client`,
```

Comment:
How about `-output-file` and `-buildpacks` flags?

Reply:
-buildpacks is set by default (running the "local" option isn't supported in the action.

Added -output-file.